docs/BigQueryTable-batchsink.md

```diff
@@ -51,6 +51,8 @@ Should only be used with the Insert operation.
 
 **Table Key**: List of fields that determines relation between tables during Update and Upsert operations.
 
+**Ordered By**: Column names and sort order that will be applied to source data during Upsert operation.
```

Comment:
This is a misleading name that doesn't really convey what this is used for. Let's change it to 'Dedupe By', with description:  

Column names and sort order used to choose which input record to update/upsert when there are multiple input records with the same key. For example, if this is set to 'updated_time desc', then if there are multiple input records with the same key, the one with the largest value for 'updated_time' will be applied.

Reply:
Done.